packages/twenty-server/src/modules/messaging/message-import-manager/drivers/gmail/services/gmail-get-message-list.service.ts

```diff
@@ -146,6 +146,7 @@ export class GmailGetMessageListService {
     return {
       messageExternalIds: messagesAddedFiltered,
       messageExternalIdsToDelete: messagesDeleted,
+      previousSyncCursor: syncCursor,
```

Comment:
I feel we should also add an interface (implements on top of the drivers to make sure they respect it)

Reply:
why do we need to return the previous syncCursor now? :)

Reply:
1 - I will implement this interface yes. Good idea to avoid future inconvenience. 
2 - Why do we need the previousSynccursor :
At this level "/messaging/message-import-manager/services/messaging-partial-message-list-fetch.service.ts" we need to know if the partial finished or not. I scratched my head to find the most elegant way to avoid mentionning folders at the messaging module level as much as possible (I want it to stay at the driver implementation).

Reply:
ok, understood